packages/leafygreen-provider/src/DarkModeContext.tsx

```diff
@@ -1,27 +1,27 @@
 import React, { createContext, PropsWithChildren, useContext } from 'react';
 import PropTypes from 'prop-types';
-import { DarkModeProps, Theme } from '@leafygreen-ui/lib';
-import type { ThemeType } from 'packages/lib/src/DarkModeProps';
 
-type DarkModeContextProps = DarkModeProps & {
-  theme?: ThemeType;
-};
+interface DarkModeContextProps {
+  globalDarkMode: boolean;
```

Comment:
I don't know if this name needs to be `globalDarkMode`. Personally, I think it's a clearer external API if it's just `darkMode`

Reply:
I named it globalDarkMode just to make the distinction clearer and because if we named it darkMode, either this darkMode or the component's darkMode prop would need an alias to avoid conflicts. Do you feel like the additional clarity would still be preferred?